cypress/e2e/CMS/$Biography/index.js

```diff
@@ -0,0 +1,146 @@
+/// <reference types="Cypress" />
+import { should } from 'chai';
+import { And, Then, Given, When } from 'cypress-cucumber-preprocessor/steps';
+
+
+When('user selects test {string}',(title) =>{
+    cy.get("input[value='Select Site Section']").should('be.visible').click({ force: true });
+
+}) 
+And('user fills out the following fields', (dataTable) => {
+    for (const { fieldLabel, value, field_name } of dataTable.hashes()) {
+        cy.get(`input[name^='${field_name}']`).as('inputField').parent().find('label').should('include.text', fieldLabel);
+        cy.get('@inputField').type(value);
+    }
+});
+    
+And('user clicks on {string} to a title section',(buttonLable) =>{
+    cy.get('#edit-field-title-add-more').contains(buttonLable).click({force:true})
+})
+And('user fills out the following fields', (dataTable) => {
+    for (const { fieldLabel, value, field_name } of dataTable.hashes()) {
+        cy.get(`input[name^='${field_name}']`).as('inputField').parent().find('label').should('include.text', fieldLabel);
+        cy.get('@inputField').type(value);
+    }
+    })
+And('user fills out {string} text area with {string}', (textFieldLabel, value) => {
+    cy.getIframeBody('iframe.cke_wysiwyg_frame.cke_reset')
+        .find('p').type(value);
+    })
+    
+    let imageSrc;
+And('user selects {int} Biography Image from the list of images', (number) => {
+    cy.get('span:contains("Biography Image")').parent().as('imageUpload').click();
+    cy.get('input[value="Select Image"]').click({ force: true });
+    cy.getIframeBody('iframe.entity-browser-modal-iframe').find("input[id^='edit-entity-browser-select-media']").eq(number -1).check();
+    cy.getIframeBody('iframe.entity-browser-modal-iframe').find("input[id='edit-submit'][value='Select image']").click({ force: true });
+});
+And('user remembers the source of selected biography image for further verification', () => {
+    cy.get('details img').then($el => {
+        imageSrc = $el[0].getAttribute('src');
+    })
+})
+And('user selects {string} checkbox',(dateDisplay)=>{
+    cy.get(`[class='fieldset-wrapper']:contains("${dateDisplay}")`).parent().find('input.form-checkbox').check({force: true})
+  
+})
+ And('user selects {string} checkbox',(dateDisplay)=>{
+    cy.get(`[class='fieldset-wrapper'] input:contains("${dateDisplay}")`).parent().find('input.form-checkbox').check({force: true})
```

Comment:
this step and the one above are the same with different implementation - which one is correct? Also there is one more of the same step in common folder - now you have 3 that are identical ( 3 times duplicated )